go.mod

```
@@ -112,3 +115,5 @@ require (
 	modernc.org/memory v1.8.2 // indirect
 	modernc.org/sqlite v1.34.5 // indirect
 )
+
+replace github.com/conductorone/dpop => ../dpop
```

golangci-lint (1.62.2)

local replacement are not allowed: github.com/conductorone/dpop

(gomoddirectives)